CHANGELOG.md

```diff
@@ -4,13 +4,24 @@ All notable changes to [Earthly](https://github.com/earthly/earthly) will be doc
 
 ## Unreleased
 
+## v0.7.21 - 2023-10-24
+
 ### Added
 - The new ARG `EARTHLY_GIT_REFS` will contain the references to the current git commit, this ARG must be enabled with the `VERSION --git-refs` feature flag. [#3341](https://github.com/earthly/earthly/pull/3341)
 - A new `--force-certificate-generation` flag for bootstrapping, which will force the generation of self signed TLS certificates even when the `--no-buildkit` flag is set.
+- Add builtin support for Git references [#3341](https://github.com/earthly/earthly/issues/2735)
```

Comment:
This description doesn't make it clear that this is a new builtin ARG, nor does it describe how to enable the new feature flag.

Upon looking at line 10, that line is actually a description for this issue; but that line incorrectly referenced the PR rather than issue. Can you change it to reference 2735 insteahd of 3341.

Reply:
I think we can actually remote line 12 since line 10 describes the PR.

Reply:
yes, but can you change it to reference 2735 instead of 3341.